ansible/roles/sdk/tasks/deploy.yml

```diff
@@ -9,25 +9,26 @@
 # Blackbox
 
 - name: make temp dir
-  local_action: shell "mktemp" "-d"
+  local_action: shell "mktemp" "-d" "{{ lookup('env', 'TMPDIR') | default('/tmp', true) }}/whisk.XXXXXXXX"
   register: tmpdir
 
+- name: copy docker sdk to dockerSkeleton in scratch space
+  local_action: copy src="{{ openwhisk_home }}/sdk/docker/" dest="{{ tmpdir.stdout }}/dockerSkeleton/"
+
 - name: build blackbox container artifact
-  local_action: shell "tar" "--exclude=build.gradle" "--transform" "s/^docker/dockerSkeleton/" "-czf" "{{tmpdir.stdout}}/blackbox-0.1.0.tar.gz" "docker" chdir="{{ openwhisk_home }}/sdk/"
+  local_action: shell "tar" "--exclude=build.gradle" "-czf" "{{ tmpdir.stdout }}/blackbox-0.1.0.tar.gz" "dockerSkeleton" chdir="{{ tmpdir.stdout }}"
 
 - name: copy blackbox container artifact to nginx
-  copy:
-    src: "{{tmpdir.stdout}}/blackbox-0.1.0.tar.gz"
-    dest: "{{ nginx_conf_dir }}"
+  local_action: copy src="{{ tmpdir.stdout }}/blackbox-0.1.0.tar.gz" dest="{{ nginx_conf_dir }}"
```

Comment:
no since it's `local_action` - good catch, will change it to a copy task as before.